core/src/lib.rs

```diff
@@ -684,34 +700,38 @@ impl Schema {
                 attribute.to_edn_value(self.get_ident(*entid).cloned()))
             .collect())
     }
+
+    fn get_raw_entid(&self, x: &NamespacedKeyword) -> Option<Entid> {
```

Comment:
This is the composition of `get_entid` and `.into()`, right?  Seems unnecessary.

Reply:
Or you could think of `get_entid` as the composition of `get_raw_entid` and `KnownEntid`!